src/qml/RosterPage.qml

```diff
@@ -57,6 +57,8 @@ Kirigami.ScrollablePage {
 				// this will update the unread message count,
 				// which will update the roster and will reset the
 				// model variable
+				print(model.jid)
+				print(kaidan.chatPartner)
```

Comment:
gives error, because you can't directly access another thread from qml ... (AFAIK)